arches/app/src/components/ControlledListManager/ItemCharacteristic.vue

```diff
@@ -0,0 +1,136 @@
+<script setup lang="ts">
+import arches from "arches";
+import { computed, ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import InputText from "primevue/inputtext";
+import { useToast } from "primevue/usetoast";
+
+import {
+    postItemToServer,
+    postListToServer,
+} from "@/components/ControlledListManager/api.ts";
+import type { Item } from "@/types/ControlledListManager.d";
+
+const props: {
+    item: Item;
+    editable: boolean;
+    field: "name" | "uri";
+    label: string;
+} = defineProps(["item", "editable", "field", "label"]);
+
+const editing = ref(false);
+const disabled = computed(() => {
+    return !props.editable || !editing.value;
+});
+
+const dirtyFormValue = ref("");
+
+const inputValue = computed({
+    get() {
+        return props.item[props.field];
+    },
+    set(newVal: string) {
+        dirtyFormValue.value = newVal;
+    },
+});
+
+const width = computed(() => {
```

Comment:
is there a way around mixing css && js for the width calc?

Reply:
+1. If you don't mind, I'll punt this to the next PR b/c I think this part of the UI is going away.